codersdk/deployment.go

```diff
@@ -1621,7 +1617,7 @@ const (
 // users to opt-in to via --experimental='*'.
 // Experiments that are not ready for consumption by all users should
 // not be included here and will be essentially hidden.
-var ExperimentsAll = Experiments{ExperimentTemplateEditor}
+var ExperimentsAll = Experiments{}
```

Comment:
The above comment is confusing to me - "should *not* be included here"? I assume any experiment included in this struct will be hidden.

Reply:
It kinda is 🤔